support/kind/kind.go

```diff
@@ -174,14 +174,11 @@ func (k *Cluster) Create(ctx context.Context, args ...string) (string, error) {
 	log.V(4).Info("Launching:", command)
 	p := utils.RunCommand(command)
 	if p.Err() != nil {
-		// Print the output data as well so that it can be useful to debug cluster bringup failures
-		var data []byte
-		b := bytes.NewBuffer(data)
-		_, err := io.Copy(b, p.Out())
+		outBytes, err := io.ReadAll(p.Out())
 		if err != nil {
 			log.ErrorS(err, "failed to read data from the kind create process output due to an error")
 		}
-		return "", fmt.Errorf("failed to create kind cluster: %s : %s: %s", p.Err(), p.Result(), b.String())
+		return "", fmt.Errorf("failed to create kind %q cluster: %s: %s: %s", k.name, p.Err(), p.Result(), string(outBytes))
```

Comment:
```suggestion
		return "", fmt.Errorf("failed to create kind cluster %q: %s: %s: %s", k.name, p.Err(), p.Result(), string(outBytes))
```

might be more readable ?